src/fileService/FileService.java

```diff
@@ -0,0 +1,66 @@
+/**
+ * 
+ */
+package fileService;
+
+import java.io.*;
+import java.util.Scanner;
+import javax.swing.JOptionPane;
+
+/**
+ * @author alial
+ *
+ */
+public class FileService {
+
+	private String fileName = "jMoSS.txt";
+
+	/*
+	 * Create method to write on file
+	 */
+	
+	// Parameters we can add them as we need 
+	public void write(String theaterLocation, String movieName, String date, String time) {
```

Comment:
file writing method needs to be generic. There are different types of data needs to be written into the file.